infrastructure/docker/builder/Makefile

```diff
@@ -21,10 +21,10 @@ IMAGENAME?=$(IMAGE_REGISTRY)/kuberpult-builder:${VERSION}
 build:
 	docker build -t $(IMAGENAME) .
 
-build-pr: build publish
+build-pr: build release
 	echo "Building on pull request"
 
-build-main: build publish
+build-main: build release
 	echo "Building on main"
 
```

Comment:
I expected a change in makefile.variables to ensure that the IMAGE_REGISTRY is set.

Reply:
I did change that, but this ofc makes it now harder to run this locally (you need to set the env IMAGE_REGISTRY as well)